iothub/device/src/InternalClient.cs

```diff
@@ -175,30 +176,24 @@ public InternalClient(IotHubConnectionString iotHubConnectionString, ITransportS
             IDelegatingHandler innerHandler = pipelineBuilder.Build(pipelineContext);
 
             if (Logging.IsEnabled)
-            {
                 Logging.Associate(this, innerHandler, nameof(InternalClient));
-            }
 
             InnerHandler = innerHandler;
 
             if (Logging.IsEnabled)
-            {
```

Comment:
Are we okay with if statements not having braces? This seems a little risky to me

Reply:
This was one area we agreed we'd reduce brace use. We have lots of cases of `if (Logging.IsEnabled) Logging.Trace(...);`

Reply:
It would certainly be nice if we didn't have to check if logging was enabled before every log statement. If only .NET was more like Java here, haha

Reply:
The logging methods don't log if it is not enabled, however  the parameters (namely string concat or interpolation) passed to the logging method are built even if the logging method doesn't log, so the cost to the client is still high (memory allocation and CPU cost) even if the statements are not logged to ETW.

Reply:
How does it work for Java?

Reply:
Hmmm. Is there an advantage of reducing the braces? It somehow looks odd to me. It will also cause of lot of these changes in every PR that touches the test files.